dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# CACHEKEY: "bionic_coq-V2020-11-24-V91"
+# CACHEKEY: "bionic_coq-V2020-11-26-V92"
```

Comment:
The number is supposed to be random not incremental btw
but we really should change the scheme to be something like "$date-(short hash of dockerfile)" (which means stop copying it in the dockerfile ofc)

Reply:
ok

Reply:
A file containing its own hash is not so easy to do ;-)